Controller/Onepage/Success.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please send an email
+ * to support@buckaroo.nl so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+
+namespace Buckaroo\Magento2\Controller\Onepage;
+
+use Magento\Sales\Api\Data\OrderPaymentInterface;
+use Buckaroo\Magento2\Model\Method\AbstractMethod;
+
+/**
+ * Override Onepage checkout success controller class
+ */
+class Success extends \Magento\Checkout\Controller\Onepage\Success {
```

Comment:
this should be done via plugins instead of overrinding the controller. If that is not possible, would be good to understand what are the blockers.

Reply:
updated my code to use plugins